### ISSUE TITLE
MISP Rest UI causing "black-holed" errors with feed/edit/<ID number> request

### ISSUE THREAD

**Issue by EdAtt** [MISP/MISP#4551]
When using the MISP Rest Query Builder UI, I try to simply update the enabled entry for a feed. But when I run it, I get a "black-holed" message page. The only data in the body section is:

```
{
"enabled": 0
}
```

What is causing this and how do I address it?

Also, the same thing occurs when I run the PyMISP code below:

```
#!/usr/bin/env python3

import urllib3
urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
from pymisp import PyMISP

#MISP UID used has role=admin
misp_key = 'AuthKey'
misp_url = 'Server URL'

misp_verifycert = False
relative_path = 'feeds/edit/2'
body = '{"enabled": 1}'

try:
    misp = PyMISP(misp_url, misp_key, misp_verifycert)
    r = misp.direct_call(relative_path, body)
except Exception as e:
    print(str(e))
if isinstance(r, dict) and 'errors' in r:
    print('ERROR: ' + str(r))
else:
    print('SUCCESS: ' + str(r))
```

Output:
`ERROR: {'name': 'The request has been black-holed', 'message': 'The request has been black-holed', 'url': '/feeds/edit/2', 'errors': ['The request has been black-holed']}`

**Comment by EdAtt** [comment]
UPDATE:  BTW, here is the error log for this activity:

`==> error.log <==

2019-04-29 19:17:29 Error: [BadRequestException] The request has been black-holed Request URL: /feeds/edit/2

Stack Trace:
#0 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Component/SecurityComponent.php(831): AppController->blackhole('auth')

#1 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Component/SecurityComponent.php(351): SecurityComponent->_callback(Object(FeedsController), 'blackHole', Array)

#2 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Component/SecurityComponent.php(255): SecurityComponent->blackHole(Object(FeedsController), 'auth', Object(AuthSecurityException))

#3 /var/www/MISP/app/Lib/cakephp/lib/Cake/Utility/ObjectCollection.php(128): SecurityComponent->startup(Object(FeedsController))

#4 /var/www/MISP/app/Lib/cakephp/lib/Cake/Event/CakeEventManager.php(243): ObjectCollection->trigger('startup')

#5 /var/www/MISP/app/Lib/cakephp/lib/Cake/Controller/Controller.php(678): CakeEventManager->dispatch(Object(CakeEvent))

#6 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(189): Controller->startupProcess()

#7 /var/www/MISP/app/Lib/cakephp/lib/Cake/Routing/Dispatcher.php(167): Dispatcher->_invoke(Object(FeedsController), Object(CakeRequest))

#8 /var/www/MISP/app/webroot/index.php(92): Dispatcher->dispatch(Object(CakeRequest), Object(CakeResponse))

#9 {main}`

**Comment by github-germ** [comment]
there was fix after 2.4.102 for this. case closed. see https://github.com/MISP/MISP/issues/4544#issuecomment-488039335
